plugin/trino-hive/src/test/sql/create-test.sql

```diff
@@ -77,7 +77,7 @@ CREATE TABLE trino_test_bucketed_by_bigint_boolean (
 )
 COMMENT 'Presto test bucketed table'
 PARTITIONED BY (ds STRING)
-CLUSTERED BY (t_bigint, t_boolean) INTO 32 BUCKETS
+CLUSTERED BY (t_bigint, t_boolean) SORTED BY (t_bigint) INTO 32 BUCKETS
```

Comment:
If `SORTED BY` is not specified, is there some default then?

Reply:
default is not to sort data, right?

Reply:
do not change this table, add a new one.

Reply:
yes, the default is unsorted, i'll add another table